spec/support/reports_helper.rb

```diff
@@ -13,6 +13,19 @@ def run_report
     expect(page).to have_button "Go", disabled: false
   end
 
+  def run_failed_report(report)
```

Comment:
I think  a single test (as you have done) is enough, therefore there's no need for this to be in a method. The code could be directly inside the test, so I'd suggest moving it there.

Reply:
I have removed the failed test from the helper to the main test. 
I have added a 2nd run (not failing) on my test.
For that, I have used the `and_call_original` to "unmock" the call, a feature I did not know, I find it very handy.